src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,179 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (head == null) {
+            head = new Node<>(null, value, null);
+            tail = new Node<>(null, value, null);
```

Comment:
here you create two different object, while in fact head and tail in list.size() == 1 are the same object